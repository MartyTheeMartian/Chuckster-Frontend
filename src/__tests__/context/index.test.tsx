import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ContextWrapper from 'context';
import { BrowserRouter as Router } from 'react-router-dom';


it('renders App component', () => {
  const result = shallow(
    <ContextWrapper>
      <Router />
    </ContextWrapper>
  );
  expect(toJson(result)).toMatchSnapshot();
});
