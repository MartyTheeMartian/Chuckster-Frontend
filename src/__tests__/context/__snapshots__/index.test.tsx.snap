// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<Fragment>
  <UserContextProvider>
    <BrowserRouter />
  </UserContextProvider>
</Fragment>
`;
